curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-form-validation-by-building-a-calorie-counter/5ddb965c65d27e1512d44d9a.md

```diff
@@ -94,7 +125,7 @@ select {
 }
 
 .output {
-  border: 2px solid black;
+  border: 2px solid var(--light-text);
```

Comment:
If you do decided to up the variable name as suggested above then it will need to be updated here too